src/main/kotlin/no/nav/familie/ba/sak/statistikk/internstatistikk/InternStatistikkService.kt

```diff
@@ -12,4 +12,6 @@ class InternStatistikkService(
     fun finnAntallFagsakerTotalt() = fagsakRepository.finnAntallFagsakerTotalt()
     fun finnAntallFagsakerLøpende() = fagsakRepository.finnAntallFagsakerLøpende()
     fun finnAntallBehandlingerIkkeErAvsluttet() = behandlingRepository.finnAntallBehandlingerIkkeAvsluttet()
+    fun finnAntallBehandlingerPerÅrsak() =
```

Comment:
Kanskje få med type her?
```suggestion
    fun finnAntallBehandlingerPerÅrsak():Map<BehandlingÅrsak, Long> =
```

Reply:
Er ikke type på noen av de andre, og er ikke så farlig kode tenker jeg.